language/use-cases/sql-code-generation/sql_code_generation_langchain.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
<p>Shouldn't this use <code>generate_sql</code> helper function which encapsulates these two above steps? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>correct, guess this was from previous notebook, changed it to invoke function.</p>